src/main.rs

```diff
@@ -77,6 +77,13 @@ fn main() {
                 .long("recursive")
                 .help("Recurse down directories if passed one"),
         )
+        .arg(
+            Arg::with_name("frames_per_second")
+                .short("f")
+                .long("frame_rate")
+                .takes_value(true)
+                .help("Play gif at the given frames rate. Units of frames per second. Default: Use delay embedded in .gif."),
```

Comment:
I believe the convention is to use dashes between words, rather than underscores.
Also, a total nitpick, but having the help on a single line when `viu --help` is called looks much better. I would remove the last two sentences.